src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_COUNT = 5;
+    private static final int MAX_NUMBER = 30;
+    private static final Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+
+        switch (random.nextInt(FIGURES_COUNT)) {
+            case 0:
+                return createRandomSquare();
+            case 1:
+                return createRandomCircle();
+            case 2:
+                return createRandomIsoscelesTrapezoid();
+            case 3:
+                return createRandomRectangle();
+            default:
+                return createRandomRightTriangle();
+
+        }
+
+    }
+
+    public Square createRandomSquare() {

```

Comment:
FigureSupplier should have only two public methods(getRandomFigure and createDefaultFigure), 
the others should be private.
All private methods should be below public methods
```suggestion
    private Square createRandomSquare() {
```